frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/AccountInterventionsHandler.java

```diff
@@ -251,14 +251,12 @@ private void submitAuditEvents(
 
     private void logAisResponse(
             AccountInterventionsInboundResponse accountInterventionsInboundResponse) {
-        if (configurationService.isAisDetailedLoggingEnabled()) {
-            LOG.info(
-                    "AIS Response: blocked: {} suspended: {} resetPassword: {} reproveIdentity: {}",
-                    accountInterventionsInboundResponse.state().blocked(),
-                    accountInterventionsInboundResponse.state().suspended(),
-                    accountInterventionsInboundResponse.state().resetPassword(),
-                    accountInterventionsInboundResponse.state().reproveIdentity());
-        }
+        LOG.info(
+                "AIS Response: blocked: {} suspended: {} resetPassword: {} reproveIdentity: {}",
+                accountInterventionsInboundResponse.state().blocked(),
+                accountInterventionsInboundResponse.state().suspended(),
+                accountInterventionsInboundResponse.state().resetPassword(),
+                accountInterventionsInboundResponse.state().reproveIdentity());
```

Comment:
Records generate `toString` by default, so you can likely have something like:

```
LOG.info("AIS Response: {}", accountInterventionsInboundResponse.state());
```

which would give
```
AIS Response: State[blocked=false, suspended=true, reproveIdentity=true, resetPassword=false]
```